src/main/java/duke/Exceptions/CommandException.java

```diff
@@ -0,0 +1,27 @@
+package duke.Exceptions;
+
+public class CommandException extends DukeException {
+    public CommandException() {
+    }
+
+    public CommandException(String message) {
+        super(message);
+    }
+
+    public CommandException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public CommandException(Throwable cause) {
+        super(cause);
+    }
+
+    public CommandException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
```

Comment:
Shouldn't enableSuppression and writableStackTrace sound like booleans?

Reply:
This are the names for the constructor of exception in the JavaDocs